easybuild/easyblocks/n/numpy.py

```diff
@@ -214,9 +241,23 @@ def get_libs_for_mkl(varname):
 
     def test_step(self):
         """Run available numpy unit tests, and more."""
-        super(EB_numpy, self).test_step()
 
-        # temporarily install numpy, it doesn't alow to be used straight from the source dir
+        if self.cfg['runtest']:
+            # Let's handle the output from the numpy test suite ourselves
+            testcmd_output, testcmd_exit_code = super(EB_numpy, self).test_step(return_testcmd_output=True)
+
+            numpy_testsuite_summary = parse_numpy_test_suite_output(testcmd_output)
+
+            if 'failed' in numpy_testsuite_summary or 'error' in numpy_testsuite_summary:
+                failure_text = "Found errors or failures in numpy testsuite output:\n %s" % numpy_testsuite_summary
+                if self.cfg['ignore_test_failures']:
+                    self.log.warning("Ignoring: ", failure_text)
```

Comment:
@ocaisa This will just be a warning in the log file, which is perhaps too easy to miss?

How about making this an actual verbose warning, via `print_warning`?